### PR TITLE
Bug-1882726 update ssh access doc

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with:
https://bugzilla.redhat.com/show_bug.cgi?id=1882726

It focuses on adding some edits and minor changes to the existing procedure and output associated with accessing a VMI by using ssh. The defect has notes regarding the changes made to the doc.

### CODE REVIEW

**Comment by lmandavi** [comment]
@phoracek Requesting your review - for details please see the defect notes.

**Comment by lmandavi** [comment]
https://bug-1882726-ssh-ext-ip-addr--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-accessing-vm-consoles.html#virt-accessing-vm-consoles-cli

**Review by phoracek — APPROVED** [review]
/lgtm

**Comment by lmandavi** [comment]
These are the only changes that were made as part of this PR in the "updates to ssh access" commit:
https://github.com/openshift/openshift-docs/pull/26724/commits/46185e4f7a4a882e5530128ecf51e1259cfa1caf
(The changes were quite minor in scope).

The other files showing changes do not belong to this PR, they were addressed via a different PR:
https://github.com/openshift/openshift-docs/pull/26298 - and this PR is presently in QE review stage.

**Comment by lmandavi** [comment]
If it makes sense to wait for this PR #26298  to be merged first, then merge this PR, we can do that.

File: modules/virt-installing-virtio-drivers-installing-windows.adoc [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-Click `Load driver`. 
+. The storage destination will not be recognized until the driver is loaded.
+Click `Load driver`.
 . The drivers are attached as a SATA CD drive. Click *OK* and browse the CD drive
- for the storage driver to load. The drivers are arranged hierarchically 
-according to their driver type, operating system, and CPU architecture. 
-. Repeat the previous two steps for all required drivers. 
-. Complete the Windows installation. 
-
+ for the storage driver to load. The drivers are arranged hierarchically
```
Comment (adellape) [inline modules/virt-installing-virtio-drivers-installing-windows.adoc:26]: I think there's an extra space at the beginning of this line, though that wouldn't affect rendering.

**Review by adellape — APPROVED** [review]
LGTM after squashing to 1 commit. :+1:

**Comment by lmandavi** [comment]
@adellape -This commit in this PR:
https://github.com/openshift/openshift-docs/pull/26724/commits/3b44c9fd72328ea401d91aeb2fa8e756f148b911

was picked up from the PR that was just merged:
https://github.com/openshift/openshift-docs/pull/26298

I might want to decouple this commit because it was already included in the merged PR? Will connect with you to get your thought on this.

**Comment by lmandavi** [comment]
CP to 4.6, 4.7. Please see my note above.

File: _topic_map.yml [hunk 2]
```diff
@@ -2546,13 +2546,20 @@ Topics:
     File: virt-controlling-vm-states
   - Name: Accessing virtual machine consoles
     File: virt-accessing-vm-consoles
+<<<<<<< HEAD
   - Name: Triggering virtual machine failover by resolving a failed node
     File: virt-triggering-vm-failover-resolving-failed-node
+=======
+>>>>>>> 9e342832d... Bug-1882726 update ssh access doc
```
Comment (adellape) [inline _topic_map.yml:2553]: Conflict here.

File: _topic_map.yml [hunk 3]
```diff
@@ -2546,13 +2546,20 @@ Topics:
...
   - Name: Installing the QEMU guest agent on virtual machines
     File: virt-installing-qemu-guest-agent
   - Name: Viewing the QEMU guest agent information for virtual machines
     File: virt-viewing-qemu-guest-agent-web
+<<<<<<< HEAD
   - Name: Managing config maps, secrets, and service accounts in virtual machines
+=======
+  - Name: Managing ConfigMaps, secrets, and service accounts in virtual machines
+>>>>>>> 9e342832d... Bug-1882726 update ssh access doc
```
Comment (adellape) [inline _topic_map.yml:2562]: Another conflict.

File: virt/virtual_machines/virt-installing-qemu-guest-agent.adoc [hunk 4]
```diff
@@ -6,6 +6,15 @@ toc::[]
 
 The xref:../../virt/virtual_machines/virt-viewing-qemu-guest-agent-web.adoc#virt-viewing-qemu-guest-agent-web[QEMU guest agent] is a daemon that runs on the virtual machine and passes information to the host about the virtual machine, users, file systems, and secondary networks.
 
+== Prerequisites
```
Comment (adellape) [inline virt/virtual_machines/virt-installing-qemu-guest-agent.adoc:9]: This heading should get an anchor/ID added.

File: virt/virtual_machines/virt-installing-qemu-guest-agent.adoc [hunk 5]
```diff
@@ -6,6 +6,15 @@ toc::[]
...
+
+* Verify that the guest agent is installed and running by entering the following command:
++
+----
+$ systemctl status qemu-guest-agent
+----
++
```
Comment (adellape) [inline virt/virtual_machines/virt-installing-qemu-guest-agent.adoc:16]: Extra `+` here can be removed.

**Review by adellape — CHANGES_REQUESTED** [review]

**Comment by vikram-redhat** [comment]
@lmandavi can this be closed?

**Comment by lmandavi** [comment]
@vikram-redhat - Yes, this GitHub issue can be closed. The content changes are being completed via a different PR. Thanks.

**Pull request closed without merging**